app/src/main/res/values/strings.xml

```diff
@@ -108,12 +108,16 @@
     <string name="confirm_favor_edit">Confirm Update</string>
     <string name="favor_cancel_success_msg">Favor successfully cancelled</string>
     <string name="favor_edit_success_msg">Favor successfully updated</string>
+    <string name="favor_complete_success_msg">Favor successfully completed</string>
     <string name="chat_message_detail_view">Start a chat</string>
     <string name="update_favor_error">Failed to update</string>
     <string name="favor_remotely_changed_msg">Favor has been updated by another!</string>
     <string name="cancel_accept_button_display">Cancel</string>
-    <string name="restart_request">re-enable</string>
+    <string name="restart_request">Re-enable</string>
     <string name="fail_edit_favor_request_view">Cannot edit favor.</string>
+    <string name="complete_favor">Complete</string>
+    <string name="wait_complete">Wait others</string>
+    <string name="favor_accept_by_oneself">you cannot accept favor posted by yourself</string>
```

Comment:
I'm not sure if this should even be possible! Maybe we should enforce that when the marker is clicked from the map view we don't display the accept view! Just a comment for next sprint

Reply:
Seeing self posted favor in accept view is rare case, it only happened once when I clicked the notification after accepter accepted my favor.